### DISCUSSION TITLE
How-to enable clipboard output for command?

### DISCUSSION THREAD

**Discussion by craziedde** [Taitava/obsidian-shellcommands#443] [Q&A] [answered]
### Obsidian debug information

Win10

### What shell are you using?

CMD.EXE

### Please describe the problem

Question in the screenshot below:
![image](https://github.com/user-attachments/assets/4e2107e7-2fae-4ce8-977f-9eb36dac71da)

@craziedde

**Comment by Taitava** [comment, marked as answer]
Hi,

here are [instructions on how to select an output channel for a shell command](https://publish.obsidian.md/shellcommands/Output+handling/Output+channels#How+to+select+output+channels+for+a+shell+command). You can select _Clipboard_ as an output channel.

- I noticed your shell command ends with `>&2`, meaning that the output is directed to `stderr` (standard error). If you keep the command as is, you need to select _Clipboard_ in _Output channel for **stderr**_.
- Normally non-error output is wanted to be directed to `stdout` (standard output) by leaving out `>&2`. If you decide to do this, then you need to select _Clipboard_ in _Output channel for **stdout**_.
